metadata/model_metadata/EleutherAI_pythia-1.4b-deduped.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "EleutherAI/pythia-1.4b-deduped",
+    "author": "EleutherAI",
+    "sha": "77f320b24ccae4aa85a5890dbb9514bd11267bb3",
+    "created_at": "2023-02-09T21:42:04+00:00",
+    "last_modified": "2023-06-08T13:03:28+00:00",
+    "private": false,
+    "gated": false,
+    "disabled": false,
+    "library_name": "transformers",
+    "tags": [
+        "transformers",
+        "pytorch",
+        "gpt_neox",
+        "text-generation",
+        "causal-lm",
+        "pythia",
+        "en",
+        "dataset:EleutherAI/the_pile_deduplicated",
+        "arxiv:2304.01373",
+        "arxiv:2101.00027",
+        "arxiv:2201.07311",
+        "license:apache-2.0",
+        "autotrain_compatible",
+        "endpoints_compatible",
+        "text-generation-inference",
+        "region:us"
+    ],
+    "pipeline_tag": "text-generation",
+    "mask_token": null,
+    "card_data": {},
+    "model_index": null,
+    "config": {
+        "architectures": [
+            "GPTNeoXForCausalLM"
+        ],
+        "model_type": "gpt_neox",
+        "tokenizer_config": {
+            "bos_token": "<|endoftext|>",
+            "eos_token": "<|endoftext|>",
+            "unk_token": "<|endoftext|>"
+        }
+    },
+    "transformers_info": {
+        "auto_model": "AutoModelForCausalLM",
+        "custom_class": null,
+        "pipeline_tag": "text-generation",
+        "processor": "AutoTokenizer"
+    },
+    "safetensors": null,
+    "training_stages": [{
+	"name": "pretraining",
+	"data": "the_pile",
+	"batch_tokens": 2097152,
+	"total_tokens_billions": 299892736000
```

Comment:
Just checking that all pythia models were trained for the same number of tokens? It seems kind of strange since usually people train larger models with more/smaller models with less but if this is not the case for pythia it's fine

Reply:
Also, this should be in billions, so this should probably be 299.89?

Reply:
Yes, all Pythia models were trained on the same number of tokens. This was intentionally done to support interpretability work by allowing people to look at each model checkpoint knowing that all checkpoints across size has seen the exact same tokens.